src/enzo/Make.mach.nasa-aitken-rome

```diff
@@ -0,0 +1,134 @@
+
+
+#-----------------------------------------------------------------------
+# Commands to run test executables
+#-----------------------------------------------------------------------
+
+
+#-----------------------------------------------------------------------
+# Install paths (local variables)
+#-----------------------------------------------------------------------
+
+LOCAL_MPI_INSTALL    = /nasa/modulefiles/sles15/mpi-hpe/mpt.2.23
+LOCAL_HDF5_INSTALL   = /nasa/modulefiles/sles15/hdf5/1.8.18_serial
+LOCAL_PYTHON_INSTALL = $(YT_DEST)
+LOCAL_COMPILER_DIR    = /nasa/modulefiles/testing/comp-intel/2020.4.304
+LOCAL_GRACKLE_INSTALL = $(HOME)/local
+
+#-----------------------------------------------------------------------
+# Compiler settings
+#-----------------------------------------------------------------------
+
+MACH_CPP       = icpc
+
+# With MPI
+
+MACH_CC_MPI    = icc
+MACH_CXX_MPI   = icpc
+MACH_FC_MPI    = ifort
+MACH_F90_MPI   = ifort
+MACH_LD_MPI    = icpc
+
+# Without MPI
+
+MACH_CC_NOMPI  = icc
+MACH_CXX_NOMPI = icpc
+MACH_FC_NOMPI  = ifort
+MACH_F90_NOMPI = ifort
+MACH_LD_NOMPI  = icpc
+
+#-----------------------------------------------------------------------
+# Machine-dependent defines
+#-----------------------------------------------------------------------
+
+MACH_DEFINES = -DLINUX -DH5_USE_16_API 
+
+#-----------------------------------------------------------------------
+# Compiler flag settings
+#-----------------------------------------------------------------------
+
+MACH_CPPFLAGS = -P -traditional
+MACH_CFLAGS   = 
+MACH_CXXFLAGS = 
+MACH_FFLAGS   = 
+MACH_F90FLAGS = 
+MACH_LDFLAGS  = 
+#,-static 
+
+#-----------------------------------------------------------------------
+# Optimization flags
+#-----------------------------------------------------------------------
+
+# *** WARNING *** WARNING *** WARNING *** WARNING *** WARNING ***
+#
+#  Use optimization -O2 or greater with PGI compilers at your own
+#  risk!  Older versions of Grid_PPMDirectEuler.C compiled with -O2
+#  led to incorrect results on many test problems, and CURRENT
+#  (-r1995) versions of ffte4X.src and ZeusSource.C lead to incorrect
+#  results for ZeldovichPancake and ExtremeAdvectionTest tests,
+#  respectively.  jobordner@ucsd.edu
+#
+# *** WARNING *** WARNING *** WARNING *** WARNING *** WARNING ***
+
+MACH_OPT_WARN        = 
+MACH_OPT_DEBUG       = -g -O0
+# Best for pleiades ivy bridge nodes.
+# See: goo.gl/A0IFMV
+MACH_OPT_HIGH        = -g -O2  -ip -ipo
+MACH_OPT_AGGRESSIVE  = -O3  -ip -ipo
+
+#-----------------------------------------------------------------------
+# Includes
+#-----------------------------------------------------------------------
+
+LOCAL_INCLUDES_MPI    = -I$(LOCAL_MPI_INSTALL)/include
+LOCAL_INCLUDES_HDF5   = -I$(LOCAL_HDF5_INSTALL)/include
+LOCAL_INCLUDES_PYTHON = -I$(LOCAL_PYTHON_INSTALL)/include/python2.7/ \
+                        -I$(LOCAL_PYTHON_INSTALL)/lib/python2.7/site-packages/numpy/core/include
```

Comment:
Python 2 is now EOL and this variable not processed further so I suggest to remove it.
```suggestion
```